sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/aio/client.py

```diff
@@ -231,7 +170,7 @@ def get_cryptography_client(self, key: Union[Key, str], **kwargs: Any) -> Crypto
         return DeletedKey._from_deleted_key_bundle(bundle)
 
     @distributed_trace_async
-    async def get_key(self, name: str, version: Optional[str] = None, **kwargs: "**Any") -> Key:
+    async def get_key(self, name: str, version: "Optional[str]" = None, **kwargs: "Any") -> Key:
```

Comment:
I like the visibility of positional `version` in `get_*` but now that's inconsistent. Anyone have an opinion?

Reply:
We decided `version` deserves to stay positional and optional.